docker/Dockerfile-fargo-master

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.6
+
+ENV GOROOT=/usr/local/go
+
+RUN go get github.com/tools/godep
+RUN go get github.com/hudl/fargo
```

Comment:
I think we can issue a PR for that and merge it as soon as Docker will add support for Windows, before that I feel  we should stick to working on all platforms solution